ui/js/directives/treeherder/resultsets.js

```diff
@@ -18,8 +18,8 @@ treeherder.directive('thResultCounts', [
                         scope.inProgress = scope.resultset.job_counts.pending +
                             scope.resultset.job_counts.running;
                         var total = scope.resultset.job_counts.completed + scope.inProgress;
-                        scope.percentComplete = total > 0 ? ((scope.resultset.job_counts.completed / total) * 100).toFixed(0) : undefined;
-                        scope.resultset.job_counts.percentComplete = scope.percentComplete;
+                        scope.percentComplete = total > 0 ?
+                            parseInt(((scope.resultset.job_counts.completed / total) * 100), 10) : undefined;
```

Comment:
The best way to do this is using floor:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/floor

Using parseInt on something gives the impression it's a string, when in fact that's already an int.